### DISCUSSION TITLE
第2节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#260] [笔记]
第2节课笔记(24班)

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/b8f05467-9a40-4171-8b35-9b317013e897

**Comment by waurblove** [comment]
https://www.bilibili.com/read/cv29432803/?spm_id_from=333.999.0.0&jump_opus=1

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day2_0.md

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135538157?spm=1001.2014.3001.5502

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135510235?spm=1001.2014.3001.5502

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135533226?spm=1001.2014.3001.5502

**Comment by yansuoyuli** [comment]
第二次作业：https://blog.csdn.net/m0_58431543/article/details/135572257

**Reply by yansuoyuli** [reply]
*笔记

**Comment by williamleo0730** [comment]
https://zhuanlan.zhihu.com/p/677638351

**Comment by JacobLiu-S** [comment]
https://www.bilibili.com/read/cv29520859/?spm_id_from=333.788.0.0 第二课笔记

**Comment by PopoData** [comment]
https://zhuanlan.zhihu.com/p/677672250

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/677671732

**Comment by aaashinshin** [comment]
https://blog.csdn.net/weixin_44254889/article/details/135575836

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135572429?spm=1001.2014.3001.5502

**Comment by flh1107** [comment]
第二课笔记：https://www.bilibili.com/read/cv29528999/
